aspnetcore/signalr/javascript-client/sample/chat.js

```diff
@@ -0,0 +1,14 @@
+﻿const logger = new signalR.ConsoleLogger(signalR.LogLevel.Information);
+const connection = new signalR.HubConnection('/chathub', logger: logger });
```

Comment:
There's a shorthand for using the console logger at a particular level. If you want `ConsoleLogger` but at a specific level, you can just write the following (no need to construct a `ConsoleLogger`)

```
const connection = new signalR.HubConnection('/chathub', logger: signalR.LogLevel.Information)
```